src/crc32c_arm64_linux_check.h

```diff
@@ -31,9 +31,10 @@ namespace crc32c {
 inline bool CanUseArm64Linux() {
 #if defined(HAVE_STRONG_GETAUXVAL) || defined(HAVE_WEAK_GETAUXVAL)
   // From 'arch/arm64/include/uapi/asm/hwcap.h' in Linux kernel source code.
+  constexpr unsigned long kHwCapPmull = 1 << 4;
```

Comment:
Very minor nit, but I think the standard (maybe just Android) constant names are HWCAP_PMULL and HWCAP_CRC32. What about adopting a constant naming convention similar to those? Maybe kHWCAP_PMULL and kHWCAP_CRC32?

Reply:
Done.

Thank you! This was bothering me, but I didn't know a good way out. I really like your suggestion!